Libraries/src/net/relinc/libraries/data/DataSubset.java

```diff
@@ -209,6 +249,15 @@ public double getDurationTrimmed(){
 		
 		return data;
 	}
+
+	public double[] getRawTimeUnsafe() {
+		// WARNING! this should only get used in select cases. `getModifiedTime()` is probably what you should be using
+		return Data.getTimeData();
+	}
+
+	public double[] getRawDataUnsafe() {
+		return Data.getData();
+	}
```

Comment:
These methods get used sparingly. One example is in Trim Data, where you want to graph the raw data alongside the modified data. 

Another option instead of having this available at all would be do disable all modifiers and then get the data, then re-enable them... idk if that's better